### PR TITLE
dnds support custom reference for hg38

### PR DESCRIPTION
**Pull request by Kdreval**
This will add support for hg38 in dnds cv

### CODE REVIEW

File: modules/dnds/1.2/src/R/run_dnds.R [hunk 1]
```diff
@@ -27,6 +27,18 @@ maf = read.table(
 
 maf = as.data.frame(maf)
 
+if (grepl("38", maf$NCBI_Build[1])) {
+    cat("Will run using the hg38 reference.\n")
+    load(snakemake@params[[3]]) # This will load the RefCDS from the path at config
```
Comment (rdmorin) [inline modules/dnds/1.2/src/R/run_dnds.R]: Isn't there a way to use named parameters here instead of referring to them positionally?

**Review by rdmorin — COMMENTED** [review]
This looks fine. I noticed while testing this that when a gene list is supplied, the covariates are ignored. This isn't really clearly documented but perhaps we should change the code to not supply the covariates when a gene list is supplied and put some comments in there explaining why so it's more clear?

**Comment by Kdreval** [comment]
> This looks fine. I noticed while testing this that when a gene list is supplied, the covariates are ignored. This isn't really clearly documented but perhaps we should change the code to not supply the covariates when a gene list is supplied and put some comments in there explaining why so it's more clear?

Will do!

File: modules/dnds/1.2/src/R/run_dnds.R (hunk 1 above)
Comment (Kdreval) [reply modules/dnds/1.2/src/R/run_dnds.R]: yep, there is - was just going with a consistent style to match other spots in this script where we refer to the snakemake objects but will swicth!

**Review by rdmorin — APPROVED** [review]

**Pull request merged**